rclcpp/include/rclcpp/node.hpp

```diff
@@ -1120,6 +1120,18 @@ class Node : public std::enable_shared_from_this<Node>
   const NodeOptions &
   get_node_options() const;
 
+  /// Manually assert that this Node is alive (for RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE).
+  /**
+   * If the rmw Liveliness policy is set to RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE, the creator
+   * of this node may manually call `assert_liveliness` at some point in time to signal to the rest
+   * of the system that this Node is still alive.
+   *
+   * \return `true` if the liveliness was asserted successfully, otherwise `false`
+   */
+  RCLCPP_PUBLIC
+  bool
```

Comment:
I think this function (and it's kin) might be a good candidate for `RCUTILS_WARN_UNUSED` (https://github.com/ros2/rcutils/blob/91c1132b0b2d223f92d132287e1bbc51adb4fd7a/include/rcutils/macros.h#L23-L27), but otherwise it's really easy to call this assuming it is working and only have it silently fail.

Reply:
Done.